testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -765,6 +845,7 @@ impl Sub for &TxStats {
             committed: self.committed - other.committed,
             expired: self.expired - other.expired,
             latency: self.latency - other.latency,
+            latency_buckets: AtomicHistogram::default().to_u64_array(),
```

Comment:
You can in theory substract histograms too, and therefore get p90 on each period, not just overall p90?

Reply:
I can do subtract P90, I thought P90 is only for overall